### PR TITLE
Automation for ZCS 5847 for Create Address List

### PR DESCRIPTION
**Pull request by piyush4360**
Added positive and negative cases for CreateAddressList Request.
Test case reference:
http://testrail01.buf.synacor.com/testrail/index.php?/cases/view/1506141f

### CODE REVIEW

File: data/soapvalidator/MailClient/AddressList/ZCS-5847_CreateAddressList.xml [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+			<t:request>
+				<AuthRequest xmlns="urn:zimbraAccount">
+					<account by="name">${acct1.name}</account>
+					<password>${defaultpassword.value}</password>
+				</AuthRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//acct:AuthResponse/acct:lifetime" match="^\d+$" />
+				<t:select path="//acct:AuthResponse/acct:authToken" set="authToken" />
+
```
Comment (chinmay15) [inline data/soapvalidator/MailClient/AddressList/ZCS-5847_CreateAddressList.xml]: remove unnecessary new line

File: data/soapvalidator/MailClient/AddressList/ZCS-5847_CreateAddressList.xml [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+						</conds>
+					</searchFilter>
+				</CreateAddressListRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:CreateAddressListResponse" />
+			</t:response>
+		</t:test>
+	</t:test_case>
+
```
Comment (chinmay15) [inline data/soapvalidator/MailClient/AddressList/ZCS-5847_CreateAddressList.xml]: remove unnecessary new line

File: data/soapvalidator/MailClient/AddressList/ZCS-5847_CreateAddressList.xml [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+	<t:test_case testcaseid="CreateAddressList_07" type="smoke"
+		bugids="ZCS-5847">
+		<t:objective>Create a new address list for all members</t:objective>
```
Comment (swapnilpingle89) [inline data/soapvalidator/MailClient/AddressList/ZCS-5847_CreateAddressList.xml]: Please make sure objective matches test scenario. Here it is creating AL for based on postal code and resource type.

**Comment by chinmay15** [comment]
please remove new lines which are not required.

**Comment by chinmay15** [comment]
please attach result

**Pull request merged**